src/components/containers/RealtimeExperimentContainer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import MonitoringChartContainer from '../containers/MonitoringChartContainer';
+
+
+class RealtimeExperimentContainer extends React.Component {
```

Comment:
Since we're only overriding the `render` method, we could make this a function.

```javascript
function RealtimeExperimentContainer() {
    // ...
}

export default withRouter(RealtimeExperimentContainer);
```

Reply:
Yea good catch. I think an earlier iteration had some lifecycle stuff.